eth_abi/encoding.py

```diff
@@ -451,7 +451,8 @@ class AddressEncoder(Fixed32ByteSizeEncoder):
     def validate_value(cls, value):
         if not is_address(value):
             raise EncodingTypeError(
-                "Value of type {0} cannot be encoded by {1}".format(
+                "Cannot encode value {0} of type {1} as an address using {2}".format(
```

Comment:
Also, I'd prefer this wording instead.  I feel like the name of the encoder implies that an address is being encoded:
```suggestion
                "Cannot encode value {0} of type {1} using {2}".format(
```

Reply:
Good call! Fixed:
https://github.com/ethereum/eth-abi/pull/102/commits/5ec58537473caf97f4c21dadab91dda8f4c8ce59